icons/xd/xd-line.svg

```diff
@@ -0,0 +1,15 @@
+<svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 128 128" style="enable-background:new 0 0 128 128;" xml:space="preserve">
+<style type="text/css">
+	.st0{fill:#DD80BC;}
+</style>
```

Comment:
Please take a look at https://github.com/devicons/devicon/blob/master/CONTRIBUTING.md#svg-standards
>Each .svg must use the fill attribute instead of using classes for colors. See here for more details.